src/main/java/de/tum/in/www1/artemis/service/connectors/athena/AthenaFeedbackSuggestionsService.java

```diff
@@ -53,6 +62,9 @@ private record ResponseDTOText(List<TextFeedbackDTO> data) {
     private record ResponseDTOProgramming(List<ProgrammingFeedbackDTO> data) {
     }
 
+    private record ResponseDTOModeling(List<ModelingFeedbackDTO> data) {
+    }
```

Comment:
```suggestion
    private record ResponseDTOModeling<FeedbackDTO>(List<FeedbackDTO> data) {
    }
```
Can we use this instead of creating an empty record for each exercise type?

Reply:
We can do so - I was pretty much following the approach for programming and text exercise but I can use the base type for modeling exercises.

Reply:
I would rather vote for refactoring this one in a follow-up PR as changing this in the codebase would also mandate a change for programming and text exercises which I feel like should not be mixed into this PR. 🤔